frontend/src/pages/Status.tsx

```diff
@@ -84,6 +85,10 @@ export function statusToIcon(
       iconColor = color.terminated;
       title = 'Run was manually terminated';
       break;
+    case NodePhase.OMITTED:
+      IconComponent = ClearIcon;
+      title = "Run was omitted and didn't execute.";
```

Comment:
I tried with a few other samples, and found that an omit state will show only when the previous node has failed. Thank you for your suggestion! I applied it with a small adjustment:

```
Run was omitted because the previous step failed.
```